SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -40,13 +41,32 @@ FORCEINLINE UClass* ResolveClass(FString& ClassPath)
 	return Class;
 }
 
+void USpatialClassInfoManager::QuitGame(UObject* a_worldContext, UClass* Class)
+{
+#if WITH_EDITOR
+	// There is no C++ method to quit the current game, so using the Blueprint's QuitGame() that is calling ConsoleCommand("quit")
+	// Note: don't use RequestExit() in Editor since it would terminate the Engine loop
+	UKismetSystemLibrary::QuitGame(a_worldContext, nullptr, EQuitPreference::Quit);
+#else
+	const bool force = false;
+	FGenericPlatformMisc::RequestExit(force);
+#endif
+}
+
 void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 {
-	checkf(IsSupportedClass(Class), TEXT("Could not find class in schema database: %s"), *Class->GetPathName());
 
 	TSharedRef<FClassInfo> Info = ClassInfoMap.Add(Class, MakeShared<FClassInfo>());
 	Info->Class = Class;
 
+	if (!IsSupportedClass(Class)) {
```

Comment:
This should be the first logic run in this function. Also please move the '{' to the next line.

Reply:
Moved `{` but can't move the conditional up, as the logic further down expects `ClassInfoMap` to have `Class` in it. Happy to chat offline if you think this could lead to problems.